src/libs/actions/BankAccounts.js

```diff
@@ -603,16 +603,29 @@ function validateBankAccount(bankAccountID, validateCode) {
 }
 
 /**
- * Set the current error message. Show Growl for errors which are not yet handled by the error Modal.
+ * Show error modal and optionally a specific error message
+ *
+ * @param {String} errorMessage The error message to be displayed in the modal's body. We show translate('companyStep.confirmModalPrompt') if not provided.
+ */
+
+function showErrorModal(errorMessage = null) {
```

Comment:
Should maybe be more specific for now `showBankAccountErrorModal()` ?

Reply:
I have my doubts about that. I feel like the developer can easily know that this function was imported from BankAccount and I like to avoid having to include that in the methods we export, but if you think is better to change it because it can avoid some confusion later.. I can do it anyway

Reply:
If you were deciding to use this pattern in other places then you'd have a generic `showErrorModal()` method when in reality it only relates to a single API command. So, I think it's better to be specific, but if you'd prefer to wait for that to become a problem them I'd understand. If you strongly disagree then disagree and we can keep discussing it.

Reply:
I don't strongly disagree, I'll rename it